src/components/tx/SignOrExecuteForm/index.tsx

```diff
@@ -261,6 +267,16 @@ const SignOrExecuteForm = ({
           </ErrorMessage>
         ) : submitError ? (
           <ErrorMessage error={submitError}>Error submitting the transaction. Please try again.</ErrorMessage>
+        ) : willExecute && isUnknown ? (
+          <ErrorMessage>
+            This Safe was created with an unsupported base contract. It should <b>ONLY</b> be used for fund recovery.
+            Transactions will execute but the transaction list may not immediately update. Transaction success can be
+            verified on the{' '}
+            <ExternalLink href={getExplorerLink(safeAddress, currentChain!.blockExplorerUriTemplate).href}>
+              {currentChain?.chainName} explorer
+            </ExternalLink>
```

Comment:
I would suggest not using `!` here but instead checking that `currentChain` exists otherwise this could lead to runtime errors.